account_manager/src/common.rs

```diff
@@ -37,3 +39,61 @@ pub fn base_wallet_dir(matches: &ArgMatches, arg: &'static str) -> Result<PathBu
         PathBuf::new().join(".lighthouse").join("wallets"),
     )
 }
+
+pub fn strip_off_newline_codes(bytes: &mut Vec<u8>) -> Vec<u8> {
+    let mut strip_off = 0;
+    for (i, byte) in bytes.iter().rev().enumerate() {
+        if *byte == NEWLINE_SLASH_N || *byte == NEWLINE_SLASH_R {
+            strip_off = i + 1;
+        } else {
+            break;
+        }
+    }
+    bytes.resize(bytes.len() - strip_off, 0);
+
+    bytes.to_vec()
+}
+
+#[cfg(test)]
+mod test {
+    use super::strip_off_newline_codes;
+
+    #[test]
+    fn test_strip_off() {
+        let expected_bytes: Vec<u8> = vec![108, 105, 103, 104, 116, 104, 111, 117, 115, 101];
```

Comment:
I would prefer these test cases used string literals and `str::as_bytes()` so that they're easier to read.

Something like:

```rust
assert_eq!("hello world".as_bytes(), &strip_off_newlines("hello world\r\n".as_bytes().to_vec()));
```